src/main/java/jsr223/cpython/utils/PythonVersionGetter.java

```diff
@@ -83,7 +83,7 @@ public String getPythonVersion(String pythonVersion) {
             result = commandOutput.toString().equals("") ? processError.toString() : commandOutput.toString();
 
         } catch (Exception e) {
-            log.warn("Could not determine python version: " + e.getMessage());
+            log.info("Could not determine python version: " + e.getMessage());
```

Comment:
this should be log.debug as the goal is to remove this message appearing each time